CHANGELOG.md

```diff
@@ -60,6 +60,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Eric Amorde](https://github.com/amorde)
   [#9612](https://github.com/CocoaPods/CocoaPods/pull/9612)
 
+* Don't crash on non UTF-8 error message
```

Comment:
nit: please add two empty spaces at the end of this line for markdown formatting.